book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] Please kindly ensure that getters and setters for @rentals should be present